docker_interop/app/build.gradle

```diff
@@ -0,0 +1,63 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java application project to get you started.
+ * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/6.8.3/userguide/building_java_projects.html
+ */
+
+plugins {
+    id "io.spring.dependency-management" version "1.0.11.RELEASE"
+    id 'application'
+}
+
+dependencyManagement {
+    dependencies {
+        dependencySet(group: 'org.hyperledger.besu.internal', version: '21.1.3') {
+            entry 'qbft'
+            entry 'core'
+            entry 'crypto'
+            entry 'common'
+        }
+
+        dependency 'org.yaml:snakeyaml:1.28'
+        dependency 'commons-io:commons-io:2.8.0'
+        dependency 'info.picocli:picocli:4.6.1'
+
+    }
+}
+
+repositories {
+    jcenter()
+    mavenCentral()
+    maven { url "https://hyperledger.jfrog.io/hyperledger/besu-maven" }
+    maven { url "https://artifacts.consensys.net/public/maven/maven/" }
+}
+
+dependencies {
+    implementation 'org.hyperledger.besu.internal:qbft'
+    implementation 'org.hyperledger.besu.internal:core'
+    implementation 'org.hyperledger.besu.internal:crypto'
+    implementation 'org.hyperledger.besu.internal:common'
+    implementation 'org.yaml:snakeyaml'
+    implementation 'info.picocli:picocli'
+
+    // Use JUnit Jupiter API for testing.
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.2'
+
+    // Use JUnit Jupiter Engine for testing.
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
```

Comment:
Are there any tests to run?

Reply:
no tests to run at the moment as the code mainly used to generate docker-compose files.